pnpm-lock.yaml

```diff
@@ -7513,7 +7488,6 @@ packages:
 
   /glob-to-regexp/0.4.1:
     resolution: {integrity: sha512-lkX1HJXwyMcprw/5YUZc2s7DrpAiHB21/V+E1rHUrVNokkvB6bqMzT0VfV6/86ZNabt1k14YOIaT7nDvOX3Iiw==}
-    dev: true
```

Comment:
I believe this is because `next` package upgrade to `14.*` changed their dependency list. This has caused some of our devDependencies to move to being regular dependencies.